tutor/plugins/openedx.py

```diff
@@ -142,3 +142,63 @@ def is_directory_mounted(image_name: str, dirname: str) -> bool:
 hooks.Filters.ENV_TEMPLATE_VARIABLES.add_item(
     ("iter_mounted_directories", iter_mounted_directories)
 )
+
+
+@hooks.Filters.LMS_WORKER_COMMAND.add(priority=hooks.priorities.HIGH)
+def _add_default_lms_worker_parameters(
+    command: list[str]
+) -> list[str]:
+    command = [
```

Comment:
We should use `append`, because we don't want to override any option set before. Actually, we don't need to add a callback function, now that we have a list command arguments:

```
hooks.Filters.LMS_WORKER_COMMAND.add_items([
   "celery",
    "worker",
    "--app=lms.celery",
    ...
])
```